src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/GetAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,173 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDatabase",
+        DefaultParameterSetName = GetByNameAndResourceGroupParameterSet),
+        OutputType(typeof(AzureSqlManagedDatabaseModel))]
+    public class GetAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<IEnumerable<AzureSqlManagedDatabaseModel>>
+    {
+        protected const string GetByNameAndResourceGroupParameterSet =
+            "GetInstanceDatabaseFromInputParameters";
+
+        protected const string GetByResourceIdParameterSet =
+            "GetInstanceDatabaseFromAzureResourceId";
+
+        protected const string GetByInputObjectParameterSet =
+            "GetInstanceDatabaseFromManagedInstanceObject";
+
+        /// <summary>
+        /// Gets or sets the name of the instance database to use.
+        /// </summary>
+        [Parameter(ParameterSetName = GetByNameAndResourceGroupParameterSet,
+            Mandatory = false,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Instance Database to retrieve.")]
+        [Parameter(ParameterSetName = GetByResourceIdParameterSet,
+            Mandatory = false,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Instance Database to retrieve.")]
+        [Parameter(ParameterSetName = GetByInputObjectParameterSet,
+            Mandatory = false,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Instance Database to retrieve.")]
+        [Alias("InstanceDatabaseName")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the Azure Sql Managed Instance
+        /// </summary>
+        [Parameter(ParameterSetName = GetByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The Azure Sql Managed Instance name.")]
+        [ValidateNotNullOrEmpty]
+        public override string ManagedInstanceName { get; set; }
```

Comment:
Since the cmdlet is getting rid of "Managed" terminology, shouldn't this just be `-InstanceName` ?

Reply:
I agree. Since New-AzureRmSqlDatabase  has parameter -ServerName, Here we should have -InstanceName.

Reply:
@jovanpop-msft @jaredmoo Ok, will change it. However, what is your take on parameter descriptions, documentation examples descriptions, etc.? Should we keep managed instance in descriptions to better describe what type of instance is the parameter about, or not?

Reply:
I would put official name Azure SQL Database Managed Instance in the main comment. and in the examples we can leave shorter unofficial names like Managed Instance or SQL Instance.

Reply:
The cmdlet's top-level description can contain the full product name "Azure SQL Database managed Instance", then each parameter and example should have just "instance". Basically when a user runs `Get-Help New-AzureSqlInstance`, it is very annoying to read the same long product name over and over again.

Reply:
@jaredmoo Thanks Jared. What is cmdlet's top-level description? Where is it defined? Running Get-Help New-AzureRmSqlInstance doesn't return cmdlet description even when called with "-Full" parameter. For now it is like this: parameter names are without "Managed", ParameterHelp is with full name (Azure SQL Database Managed Instance), and non-formal examples are with informal "managed instance"